shared_tests/analysis_book-server/source_only_reference/output.yaml

```diff
@@ -0,0 +1,376 @@
+- description: This ruleset detects logging configurations that may be problematic
+    when migrating an application to a cloud environment.
```

Comment:
@ibragins Can you please validate if all issues generated in CLI for source analysis are same as listed [here](https://docs.google.com/spreadsheets/d/1BLQiPe4i3MTr-oEciCB6iHt6Na9GQywafrrbARtOOe0/edit?gid=1309501729#gid=1309501729)